automatic/prey/README.md

```diff
@@ -5,5 +5,5 @@ Prey lets you keep track of your laptop, phone and tablet whenever stolen or mis
 
 ## Notes
 
-- To pass your `API_KEY` you can use `choco install prey -y -ia "'/API_KEY=foobarr123'`
+- To pass your `API_KEY` you can use `choco install prey -y -ia "API_KEY=foobar123`
```

Comment:
The suggestion for Chocolatey is to use `"'<string>'"` when passing things to Chocolatey from PowerShell. This has the greatest chance of passing strings in completely with quoting, and is likely what was initially intended.

```suggestion
- To pass your `API_KEY` you can use `choco install prey -y -ia "'API_KEY=foobar123'"`
```